packages/suite/src/constants/suite/experimental.ts

```diff
@@ -19,6 +21,8 @@ export const EXPERIMENTAL_FEATURES: Record<ExperimentalFeature, ExperimentalFeat
         title: 'TR_EXPERIMENTAL_PASSWORD_MANAGER',
         description: 'TR_EXPERIMENTAL_PASSWORD_MANAGER_DESCRIPTION',
         knowledgeBaseUrl: EXPERIMENTAL_PASSWORD_MANAGER_KB_URL,
+        goToRoute: 'password-manager',
```

Comment:
Please rename it to `routeName`

Reply:
Solved in 919e03efc1165779a51769ce3298867753cfe1ba